pkg/testutils/testutils.go

```diff
@@ -70,13 +70,17 @@ func CreateFakeRequestJSON(uid string,
 	}
 	switch operation {
 	case v1beta1.Create:
-		req.Request.Object = obj
+		req.Request.Object = *obj
 	case v1beta1.Update:
 		// TODO (lisa): Update should have a different object for Object than for OldObject
-		req.Request.Object = obj
-		req.Request.OldObject = obj
+		req.Request.Object = *obj
+		if oldObject != nil {
+			req.Request.OldObject = *oldObject
+		} else {
+			req.Request.OldObject = *obj
```

Comment:
@jewzaam I do this so that there is _some_ data there. In most cases right now we don't care if there's any difference, but in the test struct, `oldObject` is still optional. I'm open to making it _not_ optional in the future?